### DISCUSSION TITLE
Web development

### DISCUSSION THREAD

**Discussion by Mrivilla** [roleoroleo/yi-hack-Allwinner-v2#186] [Q&A]
Hello dears,
I would like to change the Web interface, but I'm not very familiar with web development. It seems use jquery and nodejs ? could you please confirm what technology is using and the best recommended Development Environment Interface?
Thanks.
Marcos.

**Comment by roleoroleo** [comment]
jquery is used.
Server side only cgi-script in shell script.
I don't know what's the recommended IDE, I use a simple text editor.
